app/views/DR/HomeScreen/index.js

```diff
@@ -37,14 +41,16 @@ class HomeScreen extends Component {
       recovered: 0,
       current: 0,
       refreshing: false,
+      notified: false,
+      useType: '',
```

Comment:
Any reason why we save the `userType`?

Reply:
its useType (type of use).

After a person turns out to be positive the app move to positive onbording which need the state and useType, if the useType is equal to "mySelf" that means the person is the owner of the phone and would ask you to share location with the ministry otherwise would not.